torchreid/models/efficient_net_pytcv.py

```diff
@@ -371,20 +371,22 @@ def forward(self, x, return_featuremaps=False, get_embeddings=False):
                 x = self.input_IN(x)
 
             y = self.features(x)
-            if return_featuremaps:
+            if return_featuremaps and not self.is_classification():
                 return y
 
             glob_features = self._glob_feature_vector(y, self.pooling_type, reduce_dims=False)
             logits = self.output(glob_features.view(x.shape[0], -1))
 
+            if return_featuremaps and self.is_classification():
```

Comment:
This is not compatible with Q2L and the new GCN approach. I need to infer the model and get the features only.  But, now excess operations are done. We need to think about how to get this around.

Reply:
Additional parameter to dump intermediate tensors was added